Samples/AndroidTextureOES/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+#-------------------------------------------------------------------
+# This file is part of the CMake build system for OGRE
+#     (Object-oriented Graphics Rendering Engine)
+# For the latest info, see http://www.ogre3d.org/
+#
+# The contents of this file are placed in the public domain. Feel
+# free to make use of it in any way you like.
+#-------------------------------------------------------------------
+
+# Include the CreateAndroidProj macro to create a basic android setup
+# and add_static_libs macro to define additional target static libs
+include(AndroidMacros)
+
+if(ANDROID AND OGRE_BUILD_COMPONENT_JAVA)
+    # Setup create_android_proj macro requirements
+    SET(ANDROID_MOD_NAME "AndroidTextureOES")
+    SET(JNI_PATH ".")
+    #SET(JNI_SRC_FILES "OgreActivityJNI.cpp")
+
+    SET(NDKOUT "${CMAKE_BINARY_DIR}/${ANDROID_MOD_NAME}")
+
+    SET(PKG_NAME "org.ogre.jni")
+
```

Comment:
.. `set(OGRE_ADDITIONAL_PERMISSION "<uses-permission android:name="android.permission.CAMERA" />")` here.
So only this sample requests CAMERA permissions.

Otherwise our package here would require it too: https://f-droid.org/en/packages/org.ogre.browser/